charts/zitadel/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: zitadel
 description: A Helm chart for ZITADEL
 type: application
-appVersion: "v2.49.1"
-version: 7.12.1
+appVersion: "v2.51.0"
```

```suggestion
appVersion: "v2.51.2"
```

Hi @kleberbaum 

Thanks for the update and the summary for the automation. It will help once we pick that up again.

If you'd like us to send you a small gift in return, please send us a mail to hi@zitadel.com. We will send you a form with questions about your shirt size and address.